### PR TITLE
Inter cpi leveldb 3node

### PR DESCRIPTION
**Pull request by thezhe**
Simple 3 node example for class demo and a general node class. See commit notes for more details.

### CODE REVIEW

**Comment by thezhe** [comment]
Note: my local volume is /media/niok/. This depends on everyone's local shared volume (in docker-compose.yaml)

File: CPISync/GossipUtilities.cc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    for (int i = 0; i<nodes.size(); ++i)
+    {
+        DB *db = nullptr;
+        db = Gossip::getDB(nodes[i]);
+        db->Get(ReadOptions(), key, &s);
+        cout<< nodes[i] << ": " << s << endl;
+        delete db;
+    }
+}
+}
```
Comment (wfystx) [inline CPISync/GossipUtilities.cc:68]: It'd be better if put "Node" in another file

**Review by wfystx — APPROVED** [review]
LGTM
you can put these code into src folder later, after I push my part

File: CPISync/GossipUtilities.cc (hunk 1 above)
Comment (thezhe) [reply CPISync/GossipUtilities.cc:68]: Ok, I will add those changes to my branch.

**Comment by aktrinh17** [comment]
Looks good

**Pull request merged**